auth_otp/services/otp_service.py

```
@@ -0,0 +1,341 @@
+from ..models import OTPRequest, OTPRateLimit
+from .kavenegar_service import KavenegarService
+
+logger = logging.getLogger(__name__)
+
+
+class OTPService:
+    """
+    سرویس مدیریت کامل OTP
+    """
+    
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)